{"Launchpad":"https://bugs.launchpad.net/bugs/1502138","Id":"1502138","Gerrit":"https://review.openstack.org/238505","IdGerrit":"238505","IdCommit":"98c62d2b9d44b46500cfbc2fe3fc3ac2b5955718","Files":["cinder/tests/unit/test_volume.py","cinder/volume/driver.py"],"Lines_Inseted":33,"Lines_Deleted":5,"CommitParent":"9c6d20e8a38ec78e7fc8903a757c50402faa29a6","Classification":"Bug","KeywordsTitle":"may not clean up","Description":"we do not cleanup system devices, we leave leftovers behind, fail state","KeywordsCommit":"","Comments":""}